modules/installation-configuration-parameters.adoc

```diff
@@ -1484,6 +1488,51 @@ Additional Alibaba Cloud configuration parameters are described in the following
 
 endif::alibabacloud[]
 
+ifdef::nutanix[]
+[id="installation-configuration-parameters-additional-vsphere_{context}"]
+== Additional Nutanix configuration parameters
+
+Additional Nutanix configuration parameters are described in the following table:
+
+.Additional Nutanix cluster parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.nutanix.apiVIP`
+|The virtual IP (VIP) address that you configured for control plane API access.
+|IP address.
+
+|`platform.nutanix.ingressVIP`
+|The virtual IP (VIP) address that you configured for cluster ingress.
+|IP address.
+
+|`platform.nutanix.password`
+|The password for the Prism Central user name.
+|String
+
+|`platform.nutanix.port`
+|The port used to log into Prism Central.
+|String
+
+|`platform.nutanix.PrismCentral`
+|The Prism Central domain name or IP address.
+|String
+
+|`platform.nutanix.prismElementUUID`
+|The universally unique identifier (UUID) for Prism Element.
+|String
+
+|`platform.vsphere.subnetUUID`
+|The UUID of the subnet that is linked to the Prism Element that is managing the {product-title} cluster.
+|String
+
+|`platform.nutanix.username`
+|The username for logging into Prism Central.
+|String.
+|====
+endif::nutanix[]
+
```

Comment:
@thunderboltsid i think we need to update this based on schema updates in install-config.yaml

Reply:
Done.